client/my-sites/stats/wordads/highlights-section.jsx

```diff
@@ -0,0 +1,154 @@
+	// Takes a Number, formats it to 2 decimal places, and prepends a "$".
+	// This mimics the existing behaviour. I'm assuming we only view/report
+	// on earnings in USD.
+	const formattedAmount = '$' + numberFormat( amount, 2 );
+	// Differs from previous behaviour in that we don't want "$0.00" as a result.
+	// Per design spec we'll return "$0" in this scenario.
+	// https://github.com/Automattic/wp-calypso/issues/72045
+	return formattedAmount === '$0.00' ? '$0' : formattedAmount;
+}
+
+function getHighlights( earnings ) {
+	const total = earnings && earnings.total_earnings ? Number( earnings.total_earnings ) : 0;
+	const owed = earnings && earnings.total_amount_owed ? Number( earnings.total_amount_owed ) : 0;
+	const paid = total - owed;
+
+	const highlights = [
+		{
+			heading: translate( 'Earnings', { comment: 'Total WordAds earnings to date' } ),
+			icon: <Icon icon={ payment } />,
+			amount: total,
+		},
+		{
+			heading: translate( 'Paid', {
+				comment: 'Total WordAds earnings that have been paid out',
+			} ),
+			icon: <Icon icon={ receipt } />,
+			amount: paid,
+		},
+		{
+			heading: translate( 'Outstanding amount', {
+				comment: 'Total WordAds earnings currently unpaid',
+			} ),
+			icon: <Icon icon={ tip } />,
+			amount: owed,
+		},
+	];
+	// Transform/index the data for use with React.
+	return highlights.map( ( highlight, i ) => ( {
+		id: i,
+		formattedAmount: getAmountAsFormattedString( highlight.amount ),
+		...highlight,
+	} ) );
+}
+
+function payoutNotices( earnings ) {
+	const owed =
+		earnings && earnings.total_amount_owed ? numberFormat( earnings.total_amount_owed, 2 ) : '0.00';
+	const notice = {
+		id: 'notice',
+		value: translate(
+			'Outstanding amount of $%(amountOwed)s does not exceed the minimum $100 needed to make the payment.',
+			{
+				comment: 'WordAds: Insufficient balance for payout.',
+				args: { amountOwed: owed },
+			}
+		),
+	};
+	const limit = {
+		id: 'limit',
+		value: translate(
+			'Payment will be made as soon as the total outstanding amount has reached $100.',
+			{
+				comment: 'WordAds: Payout limit description.',
+			}
+		),
+	};
+	const payout = {
+		id: 'payout',
+		value: translate(
+			'Outstanding amount of $%(amountOwed)s will be paid approximately 45 days following the end of the month in which it was earned.',
+			{
+				comment: 'WordAds: Payout will proceed.',
+				args: { amountOwed: owed },
+			}
+		),
+	};
+	return owed < 100 ? [ notice, limit ] : [ payout ];
+}
+
+function HighlightsSectionHeader( props ) {
+	const [ isTooltipVisible, setTooltipVisible ] = useState( false );
+	const infoReferenceElement = useRef( null );
+	const localizedTitle = translate( 'Totals', {
+		comment: 'Heading for WordAds earnings highlights section',
+	} );
+	if ( ! props.notices || props.notices.length === 0 ) {
+		return <h1 className="highlight-cards-heading">{ localizedTitle }</h1>;
+	}
+	return (
+		<h1 className="highlight-cards-heading">
```

Comment:
Sorry for being picky. The code here are mostly duplication of the early return above. Would it be better only to conditional show the differences?

Reply:
I initially had that but it generated an empty DIV in the DOM for some reason. If you clicked on it, it would disappear. Coding it this way resulted in cleaner DOM output. I'll check on this again to see if I was overlooking something.

Reply:
I'm still seeing this behaviour in Chrome but not in Firefox. Maybe it's just a quirk with how Chrome is reformatting it's output when browsing the DOM.